### PR TITLE
Dont rely on external api for minecraft data

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: extract_data_from_minecraft.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const downloadServer = promisify(require('minecraft-wrap').downloadServer)
+const fs = require('fs').promises
+const { execSync } = require('child_process')
+const path = require('path')
+const OUT_DIR = 'minecraft_extracted_data'
+const rm = async path => fs.rm(path, { force: true, recursive: true })
+module.exports = async () => {
+  try { await fs.mkdir(OUT_DIR) } catch (e) {}
+  try { await rm('server.jar') } catch (e) {}
+  await downloadServer('1.18', path.join(OUT_DIR, 'server.jar'))
```
Comment (rom1504) [inline extract_data_from_minecraft.js:11]: let's not hard code the version

Comment (u9g) [reply extract_data_from_minecraft.js]: done :)

**Comment by rom1504** [comment]
does it work for 1.18? (can you send the 1.18 files in a mc data pr to check ?)

**Comment by u9g** [comment]
> does it work for 1.18? (can you send the 1.18 files in a mc data pr to check ?)

Do you want just the json it generates or a blocks.json using the new generated data?

**Comment by rom1504** [comment]
the final blocks.json

**Comment by u9g** [comment]
> the final blocks.json

https://gist.github.com/u9g/4bfacc22811f0036f106dcbf722400b9

**Comment by rom1504** [comment]
https://github.com/PrismarineJS/minecraft-data/pull/484 let's see if it's good

**Comment by rom1504** [comment]
looks like not

**Comment by u9g** [comment]
> looks like not

explained it in discord

**Comment by rom1504** [comment]
merging, let's fix the drop thing in another pr

**Pull request merged**